code/modules/surgery/organs/lungs.dm

```diff
@@ -483,10 +457,10 @@
 		user.visible_message(span_notice("[user] extends [src] with [W]!"), span_notice("You use [W] to extend [src]!"), "You hear something stretching.")
 		name = "extended [name]"
 		icon_state += "-crobar" //shh! don't tell anyone i handed you this card
-		safe_oxygen_min *= 2 //SCREAM LOUDER i dont know maybe eventually
+		/*safe_oxygen_min *= 2 //SCREAM LOUDER i dont know maybe eventually
```

Comment:
What's going on here?

Reply:
this needs a rework to not be shit, will be on it